inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -1164,20 +1164,24 @@ InferenceEngine::Precision MKLDNNNode::getRuntimePrecision() const {
 
 MKLDNNNode* MKLDNNNode::NodesFactory::create(const std::shared_ptr<ngraph::Node>& op, const mkldnn::engine& eng,
                                              const MKLDNNExtensionManager::Ptr& extMgr, MKLDNNWeightsSharing::Ptr &w_cache) {
+    auto getExceptionDescWithoutStatus = [](const InferenceEngine::Exception& ex) {
```

Comment:
In general this is not clear at all why we erase some symbols from exception message.
I propose to add a comment.

Reply:
It is done to modify error message (to add info about op name instead of a mkldnn source name)
https://github.com/IvanNovoselov/openvino/blob/a10b816cb240930f19bcc9b38d4d34ed45e73f91/inference-engine/src/mkldnn_plugin/mkldnn_node.cpp#L1222-L1228

Reply:
It's even more confusing.
I know this is not directly related to you changes, but 
Why do we have to remove something from the initial error message?
And if there is good reason to do so we need to add a comment (maybe even with example)
Otherwise it is absolutely not clear for the person reading this code for the first time.

Reply:
I agree, a detailed comment with example is added.